src/Microsoft.AspNet.Mvc.Razor/Compilation/CompilerCacheResult.cs

```diff
@@ -11,31 +13,35 @@ namespace Microsoft.AspNet.Mvc.Razor.Compilation
     public class CompilerCacheResult
     {
         /// <summary>
-        /// Result of <see cref="ICompilerCache"/> when the specified file does not exist in the
-        /// file system.
-        /// </summary>
-        public static CompilerCacheResult FileNotFound { get; } = new CompilerCacheResult();
-
-        /// <summary>
         /// Initializes a new instance of <see cref="CompilerCacheResult"/> with the specified
         /// <see cref="CompilationResult"/>.
         /// </summary>
         /// <param name="compilationResult">The <see cref="Compilation.CompilationResult"/> </param>
-        public CompilerCacheResult(CompilationResult compilationResult)
+        public CompilerCacheResult(CompilationResult compilationResult, IList<IChangeToken> expirationTokens)
```

Comment:
nit: Should we also just add a constructor for the no-token case (precompilation)?